mcr.cpp

```diff
@@ -26,19 +30,24 @@ int main(){
                 cout << "Player 1: ";
             else
                 cout << "Player 2: ";
-            cout << "Which cell to mark? i:[1..3], j:[1..3]: "; 
+            cout << "Which cell to mark? i:[0..2], j:[0..2]: "; 
             cin >> i >> j;
```

Comment:
What if the user input i = 4, j = 4?

Reply:
resolved, please reference to commit 
"[fix: add input type check for the row and column](https://github.com/crapper/modern-code-review/pull/1/commits/c7828b25c6fe158257d4e3001b840bedfa1c8151)"